mods/ra/chrome/ingame.yaml

```diff
@@ -27,3 +27,8 @@ Container@INGAME_ROOT:
 				Container@PERF_ROOT:
 		Container@MENU_ROOT:
 		TooltipContainer@TOOLTIP_CONTAINER:
+		Label@MISSION_TEXT:
+			X: 500
+			Y: 22
+			Height: 25
+			Font: Bold
```

Comment:
`UseContrast: true` might make this look better and match the other text displays:

![image](https://cloud.githubusercontent.com/assets/756669/7724939/4134c7b4-fef6-11e4-8357-d3b9b9372a7b.png)